tests/FSharp.Compiler.Private.Scripting.UnitTests/FSharpScriptTests.fs

```diff
@@ -211,16 +211,6 @@ System.Configuration.ConfigurationManager.AppSettings.Item "Environment" <- "LOC
         Assert.Equal(errors.Length, 1)
         Assert.Equal(errors.[0].ToString(), error0)
 
-    [<Theory>]
-    [<InlineData("""#i "nuget:foo" """,
-                 "input.fsx (1,1)-(1,15) interactive error Invalid URI: The format of the URI could not be determined.")>]
-    member _.``Script with #i and forgot to add quotes``(code, error) =
```

Comment:
I am not entirely sure what this test was supposed to test based on the name. Now this will result in a different error, related to non-existing directory. I will add explicit test for that in another commit.